gulpfile.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
We should not be using both gulp and grunt...

Reply:
gulp is used for the generator. Grunt is used for the component generate by the generator

Reply:
I vote neither :)